frontend/src/lib/lemon-ui/LemonInputSelect/LemonInputSelect.tsx

```diff
@@ -368,20 +366,21 @@ export function LemonInputSelect({
         )
     }, [mode, values, allowCustomValues, itemBeingEditedIndex, inputValue])
 
+    // Positioned like a placeholder but rendered via the suffix since the actual placeholder has to be a string
     const countPlaceholder = useMemo(() => {
-        if (displayMode !== 'count' || mode !== 'multiple') {
-            return placeholder
-        }
-        const selectedCount = values.length
-        const totalCount = options.length
-
-        if (selectedCount === 0) {
-            return `None selected`
-        } else if (selectedCount === totalCount) {
-            return `All ${totalCount} selected`
+        if (displayMode !== 'count' || mode !== 'multiple' || inputValue) {
+            return null
         }
-        return `${selectedCount}/${totalCount} selected`
-    }, [displayMode, mode, values, options, placeholder])
+        return values.length === 0 ? (
+            <span className="-ml-2 text-muted">None selected</span>
+        ) : (
+            <LemonSnack className="-ml-2">
+                {values.length === options.length
+                    ? `All ${options.length} selected`
+                    : `${values.length}/${options.length} selected`}
+            </LemonSnack>
```

Comment:
I agree with Zach's point in the sense of a placeholder looking like an empty state, however a snack isn't a great solution here, since this pattern should be used when there are multiple snacks at once - not just one catch-all.
What about something simple in between: the empty is just the same text, not a placeholder and not a snack either
```suggestion
            {values.length === options.length
                ? `All ${options.length} selected`
                : `${values.length}/${options.length} selected`}
```